shakedown/dcos/package.py

```diff
@@ -257,3 +257,32 @@ def remove_package_repo(
 
     cosmos = _get_cosmos()
     return cosmos.remove_repo(repo_name)
+
+def delete_zk_node(
+    zk_path
+):
+    """Delete Zookeeper node
+        :param zk_path: path in zookeeper to remove (Without the leading /)
+    """
+    return delete_framework_cruft(None, None, zk_path)
+
+def delete_framework_cruft(
+    role,
+    principal,
+    zk_path
+):
+    """Delete framework cruft in mesos after an uninstall such as zk nodes
+
+        :param role: name of the role used for reservations
+        :param principal: name of the principal used for reservations or security
+        :param zk_path: path in zookeeper to remove (Without the leading /)
+    """
+    janitor_cmd = 'docker run mesosphere/janitor:verbose /janitor.py'
```

Comment:
Yep, I'm taking a look at this.

> take the relevant functionality from `janitor.py`

For posterity, `janitor.py` lives here: https://github.com/mesosphere/framework-cleaner